bahis-kobo/kpi/settings/dev.py

```diff
@@ -0,0 +1,42 @@
+import os
+from kobo.settings.dev import *
+
+
+SESSION_COOKIE_NAME = 'sharful'
+SESSION_COOKIE_DOMAIN = '.sharful.com'

```

Comment:
This looks wrong?

Reply:
> This looks wrong?

yes. it should be in the environment file. I am making the change. thanks

Reply:
@ChasNelson1990 Fixed